src/Runner.Worker/ActionRunner.cs

```diff
@@ -181,7 +181,7 @@ public async Task RunAsync()
             else
             {
                 var templateEvaluator = ExecutionContext.ToPipelineTemplateEvaluator();
-                inputs = templateEvaluator.EvaluateStepInputs(Action.Inputs, ExecutionContext.ExpressionValues, ExecutionContext.ExpressionFunctions);
```

Comment:
The context of HashFilesFunction is decided based on the inputs here

Reply:
Passing down ExecutionContext (like we do for StepIf, like `success()` makes it available in the function later)